app/service/CountryService.scala

```diff
@@ -29,6 +29,8 @@ trait CountryService {
 
   def find(code: String)(implicit messages: Messages): Option[Country]
 
+  def isWelsh(implicit messages: Messages) = messages.lang.code == "cy"
```

Comment:
leave this function undefined or remove it from the implementation please - don't override it with the same function